API/testrunner/devices/device_base.py

```diff
@@ -67,10 +67,10 @@ def login(self):
                 self.channel.exec_command('\n\n')
                 self.channel.exec_command('cd /test')
 
-                if self.env['info']['coverage'] and self.app == 'iotjs':
+                if self.app == 'iotjs':
                     # Set up the wifi connection.
-                    wifi_name = utils.get_environment('ARTIK_COV_WIFI_NAME')
-                    wifi_pwd =  utils.get_environment('ARTIK_COV_WIFI_PWD')
+                    wifi_name = utils.get_environment('ARTIK_WIFI_NAME')
+                    wifi_pwd =  utils.get_environment('ARTIK_WIFI_PWD')
 
```

Comment:
These are not needed in case of stm32f4dis target.

Reply:
Fixed.